mapbox/libjava-services/src/test/fixtures/directions_v5_fixtures_rotary.json

```diff
@@ -190,7 +190,8 @@
               "duration": 150.6,
               "rotary_name": "Dupont Circle Northwest",
               "distance": 189.9,
-              "name": "P Street Northwest"
+              "name": "P Street Northwest",
+              "exit": "M Street"
```

Comment:
Was the fixture updated using `make directions-fixtures` or was the value manually placed in here? If the latter, it will be difficult later down the road when we need to update the fixtures again and the `exit` value's missing resulting in a failing test. @1ec5 Do you guys have a fixture which includes the step exit?